client/pages/BoardsPage.jsx

```diff
@@ -1,8 +1,33 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
+import axios from 'axios';
+
+function BoardsPage({user}) {
+
+  const [boardName, setBoardName] = useState('');
+
+  console.log('in boards page, user is: ', user);
+  const username = user.username;
+  useEffect(() => {
+    // here's where we want to get all boards from db
+    // axios.get();
+  }, []);
```

Comment:
Do we plan to implement this and if so, how would it be used in the component?

Reply:
I think @clark-pang and I talked about getting this implemented tomorrow, still figuring out the details.

Reply:
Yes, I did want to do this. It'd be a get request to the /boards endpoint to get all of the board data for the current user. Then we would simply display all boards as links that would route to the corresponding BoardPage. I've added it to the Trello board and have assigned myself to it. It might require backend refactoring as well which I'm willing to take on